PurrSoft/PurrSoft.Api/Controllers/VolunteerController.cs

```diff
@@ -0,0 +1,88 @@
+﻿using AlbumStore.Application.Commands.VolunteerCommands;
+using AlbumStore.Application.Queries.VolunteerQueries;
+using AlbumStore.Application.QueryOverviews;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using PurrSoft.Api.Controllers.Base;
+using PurrSoft.Application.Common;
+using System.Net;
+
+namespace PurrSoft.Api.Controllers;
+
+[Route("api/[controller]")]
+[ApiController]
+public class VolunteerController : BaseController
+{
+    [HttpGet("")]
+    [Authorize(AuthenticationSchemes = "Bearer")]
+    [ProducesResponseType(typeof(List<VolunteerOverview>), (int)HttpStatusCode.OK)]
```

Comment:
Think what roles need access to every method in controller? Can anyone authenticated get them? Can guests them? As in it is anonymous. Especially in the case of getters.